dist/main/index.js

```diff
@@ -5064,16 +5064,24 @@ const getPackagesPaths = async (rushRootPath) => {
   for (const project of rushJson.projects) {
     paths.push({
       packageName: project.packageName,
-      packagePath: join(rushRootPath, project.projectFolder)
+      packagePath: join(rushRootPath, project.projectFolder),
+      packageVersionPolicy: project.versionPolicyName
     })
   }
   return paths
 }
 
 const getAllChanges = async ({ rushChangePath, packagePaths, options = {} }) => {
   // Identify changed packages from change logs
-  const changedPackages = await getPackagesFromChanges(rushChangePath)
-
+  var changedPackages = await getPackagesFromChanges(rushChangePath)
+  // Filter projects by version policy
+  console.log("versionPolicy: ", options.versionPolicy)
+  if (options.versionPolicy != "") {
+    const versionPolicyPackages = packagePaths.filter(project => project.packageVersionPolicy == options.versionPolicy).map(project => project.packageName)
+    console.log("versionPolicyPackages :", versionPolicyPackages)
+    changedPackages = changedPackages.filter(project => versionPolicyPackages.includes(project))
+    console.log("changedPackages: ", changedPackages)
```

Comment:
I dont think we need this as Rush Changed projects already put output in console. this avoids duplication

Reply:
done